### PR TITLE
Refactoring interop-related constants and shared files to follow the coding conventions

### PR DESCRIPTION
**Pull request by Marusyk**
Made changes base on this [comment](https://github.com/dotnet/corefx/pull/33835#issuecomment-445430688)

Thank you @jkotas

Please review

### CODE REVIEW

File: src/System.Diagnostics.EventLog/src/System/Diagnostics/NativeMethods.cs [hunk 1]
```diff
@@ -2,22 +2,11 @@
...
-
-        public const int WAIT_OBJECT_0 = 0x00000000;
-        public const int WAIT_TIMEOUT = 0x00000102;
-        public const int WAIT_ABANDONED = 0x00000080;
-        public const int SEEK_READ = 0x2;
-        public const int FORWARDS_READ = 0x4;
-        public const int BACKWARDS_READ = 0x8;
-        public const int ERROR_EVENTLOG_FILE_CHANGED = 1503;
-        public const int ERROR_FILE_NOT_FOUND = 2;
+        internal const int ERROR_EVENTLOG_FILE_CHANGED = 1503;
```
Comment (jkotas) [inline src/System.Diagnostics.EventLog/src/System/Diagnostics/NativeMethods.cs]: These can be moved either right next to the API the returns this error (like this: https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.FormatMessage.cs#L17) or to big file all error codes (src\Common\src\CoreLib\Interop\Windows\Interop.Errors.cs). We are using both styles - unclear which one is better.

Comment (Marusyk) [reply src/System.Diagnostics.EventLog/src/System/Diagnostics/NativeMethods.cs]: What about to move that to https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/kernel32/Interop.FormatMessage_SafeLibraryHandle.cs ?

File: src/Common/src/Interop/Windows/kernel32/Interop.Constants.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal const int WAIT_TIMEOUT = 0x00000102;
```
Comment (jkotas) [inline src/Common/src/Interop/Windows/kernel32/Interop.Constants.cs]: You can delete `partial class WaitOption` in the other file, and replace its uses with this definition.

File: src/System.Diagnostics.EventLog/src/System/Diagnostics/NativeMethods.cs (hunk 1 above)
Comment (jkotas) [reply src/System.Diagnostics.EventLog/src/System/Diagnostics/NativeMethods.cs]: There error codes do not belong to `Interop.FormatMessage_SafeLibraryHandle`.
They are not related in any way to the `FormatMessage` API.

I think just add them to `src\Common\src\Interop\Windows\Interop.Errors.cs` instead.

(The other option would be to add them to `src\Common\src\Interop\Windows\advapi32\Interop.ReadEventLog.cs`.)

File: src/Common/src/Interop/Windows/Interop.Errors.cs [hunk 3]
```diff
@@ -83,5 +83,6 @@ internal partial class Errors
         internal const int ERROR_RESOURCE_LANG_NOT_FOUND = 0x717;
         internal const int EFail = unchecked((int)0x80004005);
         internal const int E_FILENOTFOUND = unchecked((int)0x80070002);
+        internal const int ERROR_EVENTLOG_FILE_CHANGED = 1503;
```
Comment (jkotas) [inline src/Common/src/Interop/Windows/Interop.Errors.cs]: Nit: To match the style of the rest of the file, this should be: 

`internal const int ERROR_EVENTLOG_FILE_CHANGED = 05DF;`

And it should be in the list between `ERROR_CLASS_ALREADY_EXISTS` and `ERROR_TRUSTED_RELATIONSHIP_FAILURE`.

**Comment by jkotas** [comment]
Thanks for cleaning this up

**Pull request merged**